script.js

```diff
@@ -0,0 +1,442 @@
+        {
+          if(value == "sqrt(")
+          {
+          textInput.value += value;
+          const w = Array.from(textInput.value);
+          w.splice(w.length-5,0,"*")
+          const result = w.join('');
+          textInput.value = result;
+          return;
+          }
+
+          if(value == "e")
+          {
+          textInput.value += value;
+          const w = Array.from(textInput.value);
+          w.splice(w.length-1,0,"*")
+          const result = w.join('');
+          textInput.value = result;
+          return;
+          }
+
+          
+          if(value == "pi")
+          {
+          textInput.value += value;
+          const w = Array.from(textInput.value);
+          w.splice(w.length-2,0,"*")
+          const result = w.join('');
+          textInput.value = result;
+          return;
+          }
+        }
+
+          textInput.value += value;
+          const w = Array.from(textInput.value);
+          w.splice(w.length-4,0,"*")
+          const result = w.join('');
+          textInput.value = result;
+          return;
+        }
+
+        if(value == "(")
+        {
+        if(s[s.length-1] === "0" || s[s.length-1] === "1"  || s[s.length-1] === "2"  || s[s.length-1] === "3"  || s[s.length-1] === "4"  || s[s.length-1] === "5"  || s[s.length-1] === "6" || s[s.length-1] === "7" || s[s.length-1] === "8" || s[s.length-1] === "9" || s[s.length-1] === ")" ||s[s.length-1] === "i" || s[s.length-1] === "e" )
+        {
+          textInput.value += value;
+          const w = Array.from(textInput.value);
+          w.splice(w.length-1,0,"*")
+          const result = w.join('');
+          textInput.value = result;
+          return;
+        }
+      }
+
+    textInput.value += value;  
+}
+
+function EraseContent() {
+    const destinationDiv = document.getElementById('textInput');
+    const destinationDiv1 = document.getElementById('answer');
+    destinationDiv.value = "";
+    destinationDiv1.value = "";
+    const textInput1 = document.getElementById('answer');
+    textInput1.style.display = "none";
+    const textInput = document.getElementById('textInput');
+    textInput.style.height = "100%";
+    textInput.style.fontSize = "30px";
+}
+
+function removeLastCharacter() {
+  const inputElement = document.getElementById("textInput");
+  const currentValue = inputElement.value;
+
+  if (currentValue.length > 0) 
+  {
+    const newValue = currentValue.slice(0, -1);
+    inputElement.value = newValue;
+  }
+}
+
+function precedence(operator) {
+  if (operator === '+' || operator === '-') 
+    return 1;
+
+  if (operator === '*' || operator === '/') 
+    return 2;
+
+  if (operator === 'sin' || operator === 'cos' || operator === 'tan' || operator === 'sqrt') 
+    return 3;
+
+  if (operator === '^') 
+    return 4;
+  
+  return 0;
+}
+
+function isOperator(token) {
+  return ['+', '-', '*', '/', 'sin', 'cos', 'tan', 'sqrt','^'].includes(token);
```

Comment:
create a constant array for this `['+', '-', '*', '/', 'sin', 'cos', 'tan', 'sqrt','^']` and use it here